st2common/bin/st2ctl

```diff
@@ -17,6 +17,13 @@
 COMPONENTS="st2actionrunner st2api st2stream st2auth st2garbagecollector st2notifier st2resultstracker st2rulesengine st2sensorcontainer st2chatops st2timersengine st2workflowengine st2scheduler mistral"
 ST2_CONF="/etc/st2/st2.conf"
 
+# Ubuntu Bionic packages ship without Mistral
+SUBTYPE=$(lsb_release -a 2>&1 | grep Codename | grep -v "LSB" | awk '{print $2}')
```

Comment:
There are 2 things here:
- `lsb_release --codename --short` should output `bionic`
- `lsb_release` is not available under EL systems

How about simple `py` version check in `/opt/stackstorm/st2/bin`. If it's `py2` - work as expected, if `py3` - skip mistral? This will also work long-term for other py3 distros we'll add in future (EL8?).

Reply:
05964b4, again, I'm fine either way.